.circleci/config.yml

```diff
@@ -104,7 +104,7 @@ commands:
             pushd ..
             git clone https://chromium.googlesource.com/external/gyp.git
             pushd gyp
-            python setup.py install
+            python3 setup.py install
```

Comment:
in https://github.com/mogemimi/pomdog/wiki/How-to-Install-GYP i saw "Python 2.7+ (GYP does not support Python 3, at the moment)" , I guess that's old?

Reply:
Yeah this was actually fixed a few months ago.

Reply:
https://chromium-review.googlesource.com/c/external/gyp/+/1639564

Reply:
nice!